tribeiro/Rotator/basic_rotator_test.ipynb

```diff
@@ -219,7 +223,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.2"
+   "version": "3.6.8"
```

Comment:
Why this change? It would certainly be better to run with Python 3.7 if we can get it.